euphony/src/main/java/co/euphony/rx/EuRxManager.java

```diff
@@ -96,14 +107,24 @@ private boolean listenOnNative() {
 	}
 
 	public boolean listen() {
-		if(rxEngineType == RxEngineType.EUPHONY_JAVA_ENGINE)
-			return listenOnJava();
+		return listen(0, () -> {
+		});
```

Comment:
here is void lambda.
how about using `null` instead of empty lambda?
```suggestion
		return listen(0, null);
```

Reply:
I changed! [6f6643e](https://github.com/euphony-io/euphony/pull/229/commits/6f6643e3fcd2ad8597eaa80f5dc495ef4fc139f6)